database/migrations/2018_06_04_128900_create_app_message_table.php

```diff
@@ -3,7 +3,7 @@
 use Illuminate\Database\Migrations\Migration;
 use Illuminate\Database\Schema\Blueprint;
 
-class CreateAPPMESSAGETable extends Migration {
+class CreateAppMessagetable extends Migration {
```

Comment:
Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.